README.md

```diff
@@ -72,7 +72,7 @@ on the `sbot.box2` namespace:
   - `key` must be a buffer. The key can then be used as a "recp" to encrypt messages to the group. Note that the keys are not persisted in this module.
   - `scheme` _String_ - scheme of that encryption key (optional, there is only one option at the moment which we default to)
   - `root` _MessageId_ the id of the `group/init` message
-- `listGroupIds(cb) => [groupIds]`: Lists all groupIds whose messages you're able to decrypt. Returns a promise if cb isn't provided.
+- `listGroupIds({ live }, cb) => [groupIds]`: Lists all groupIds whose messages you're able to decrypt. If `live` is true then the cb instead returns a pull stream with all previous and future group ids. Returns a promise if cb isn't provided.
```

Comment:
Like my comment in the ssb-keyring PR, this API should also return the same consistent type, i.e. always pull-stream.

Reply:
yeah actually using this in tribes2 felt very weird

Reply:
done